src/test/java/api/support/fixtures/RequestsFixture.java

```diff
@@ -206,4 +206,13 @@ public Response attemptMove(MoveRequestBuilder requestToBuild)
       200, "request-queue-request")),
       Function.identity() ,"requests").value();
   }
+
+  public void replace(UUID id, RequestBuilder builder)
```

Comment:
What is the value of this wrapper? Is it used in a situation where the request client isn't available?

Reply:
Yes, it was intended for that situation, but I found that client is available in the place where fixture is used. Removed the wrapper.